identity-service/src/routes/healthCheck.js

```diff
@@ -4,11 +4,107 @@ const { handleResponse, successResponse, errorResponseServerError } = require('.
 const { sequelize } = require('../models')
 const { getRelayerFunds } = require('../txRelay')
 const Web3 = require('web3')
+const web3 = new Web3(new Web3.providers.HttpProvider('https://poa-gateway.audius.co'))
+
 const axios = require('axios')
 
 let notifDiscProv = config.get('notificationDiscoveryProvider')
 
+/**
+ * Helper method to sanitize query params
+ * @param {} startBlock startBlock value
+ * @param {*} endBlock endBlock value
+ */
+const validateQueryParams = (startBlock, endBlock) => {
```

Comment:
i don't think this needs to be a helper function since it's never used by any other route. you could just put it in the route and it could be simplified a bit